sampleapp/README.md

```diff
@@ -88,9 +88,9 @@ err := appClient.Mount().CreateMount(&mountOptions)
 
 Note: 
 
-All the models for Management Structs & Variables are defined [here](https://github.com/cohesity/management-sdk-go/models)
+All the models for Management Structs & Variables are defined [here](https://github.com/cohesity/management-sdk-go/tree/master/models)
 
-All the models for App Structs & Variables are defined [here](https://github.com/cohesity/app-sdk-go/models)
+All the models for App Structs & Variables are defined [here](https://github.com/cohesity/management-sdk-go/tree/master/models)
```

Comment:
Line 93 should point to app-sdk-go. https://github.com/cohesity/app-sdk-go/tree/master/models